ui/app/mainui/AppMain.qml

```diff
@@ -354,11 +355,21 @@ Item {
                             onTriggered: popups.openCommunityProfilePopup(appMain.rootStore, model, communityContextMenu.chatCommunitySectionModule)
                         }
 
+                        MuteChatMenuItem {
+                            enabled: !model.muted
+                            title: qsTr("Mute Community")
+                            onMuteTriggered: {
+                                communityContextMenu.chatCommunitySectionModule.setCommunityMuted(interval)
+                                communityContextMenu.close()
+                            }
+                        }
+
                         StatusAction {
-                            text: model.muted ? qsTr("Unmute Community") : qsTr("Mute Community")
-                            icon.name: model.muted ? "notification-muted" : "notification"
+                            enabled: model.muted
+                            text: qsTr("Unmute Community")
+                            icon.name: "notification-muted"
                             onTriggered: {
-                                communityContextMenu.chatCommunitySectionModule.setCommunityMuted(!model.muted)
+                                communityContextMenu.chatCommunitySectionModule.setCommunityMuted(Constants.MutingVariations.Unmuted)
```

Comment:
Nice but these mute/unmute menu items (perhaps the whole context menu?) just seems duplicated here and CommunitiesListPanel.qml. Maybe extract it to some share component?

Reply:
Yeah I thought about it, but there may be another list of menu items.